pom.xml

```diff
@@ -133,6 +133,8 @@
     <!-- org.eclipse.tycho plugin version -->
     <version.org.eclipse.tycho>0.24.0</version.org.eclipse.tycho>
     <version.org.jboss.tools.tycho-plugins>0.23.1</version.org.jboss.tools.tycho-plugins>
+    <!-- Override plexus Archiver with version 3.0.3 as older don't work on JDK9. Can be removed after upgrade to jboss-parent:20+ -->
+    <version.plexus.archiver>3.0.3</version.plexus.archiver>
```

Comment:
To follow our conventions, the property name should be `version.org.codehaus.plexus.plexus-archiver`. I will fix it.